src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -150,8 +150,10 @@ REGISTER_DECLARATION(Register, rlocals,   r24);
 REGISTER_DECLARATION(Register, rbcp,      r22);
 // Dispatch table base
 REGISTER_DECLARATION(Register, rdispatch, r21);
-// Java stack pointer
+// Java interpreter stack pointer
 REGISTER_DECLARATION(Register, esp,      r20);
+// Sender's SP while in interpreter
+REGISTER_DECLARATION(Register, r19_sender_sp, r19);
```

Comment:
I don't think rmonitors above is used anywhere, perhaps you could reuse r25? Or at least delete the definition of rmonitors while you're editing this.